scripts/insert_into_appointments.sql

```diff
@@ -1 +1,6 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(date,time,comment,doctor_id,patient_id) VALUES
+  ('2016-08-12', '10am','5','9' ),
+  ('2016-08-17', '1pm','7','6' ),
+  ('2016-08-12', '10am','7','9' ),
+  ('2016-08-17', '1pm','5','8' ),
```

Comment:
Very clever, but no. 

You have knowledge about the doctors' and patients' id numbers because you have direct access to the data file, which also happens to be very short. The idea here is to insert appointments using `SELECT`-`FROM`-`WHERE` clauses. Here is 1 solution:

```sql
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT '20160812', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hilton'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20160817', '1300',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Talley')
  UNION
  SELECT '20160812', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20160817', '1300',
   (SELECT id FROM doctors WHERE surname = 'Hilton'),
   (SELECT id FROM patients WHERE surname = 'Hodge')
;
```